### ISSUE TITLE
Increase Celery time limit for git-extraction of source files

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-server#10901]
https://sentry.prod.mozaws.net/operations/olympia-prod/issues/5337688/

```
SoftTimeLimitExceeded: SoftTimeLimitExceeded()
(3 additional frame(s) were not displayed)
...
  File "olympia/amo/decorators.py", line 108, in wrapper
    return f(*args, **kw)
  File "olympia/versions/tasks.py", line 160, in extract_version_source_to_git
    version=version, author=author)

SoftTimeLimitExceeded: SoftTimeLimitExceeded()
```

**Comment by EnTeQuAk** [comment]
That's quite problematic, the soft time limit is set to 30 minutes and I noticed that this primarily happens on source uploads.

Maybe that's related to the amount of small files in source files, compared to regular XPI files. Still investigating.

**Comment by EnTeQuAk** [comment]
Looking at one example, it sounds very likely. Source files (well, others too) may for example contain `node_modules` folder and thus contain huge amounts of small files.

**Comment by willdurand** [comment]
According to our new Grafana dashboard and metrics collected during the git extraction as well as the previous comments, this issue is clearly about increasing the time limit for git-extraction of source files. That's why I updated the issue.

We do not do anything with the source files in git yet so I unassigned myself and this issue will likely be fixed when working on #14070.
